src/client/interface.ts

```diff
@@ -33,21 +23,21 @@ export interface IssuerConfigInterface {
     onChain: boolean;
     title?: string;
     image?: string;
+    filters?: {};
 }
 
 export interface NegotiationInterface {
     type: string;
     issuers?: (OnChainTokenConfig | OffChainTokenConfig)[];
-    options: {
-        overlay: UIOptionsInterface;
-        filters: {};
-    };
-    ipfsBaseUrl?: string;
+    uiOptions?: UIOptionsInterface;
     autoLoadTokens?: number | boolean;
     autoEnableTokens?: boolean;
-    autoPopup?: boolean;
     messagingForceTab?: boolean;
     safeConnectOptions?: SafeConnectOptions;
+    unSupported?: {
+        config: BrowserDataInterface,
```

Comment:
@nicktaras , "config" property name is not obvious. can we rename it to "browserList" ?

Reply:
Thanks @oleggrib, to make this clear to the web developer, this will be updates to "unSupportedUserAgent"